general/README.md

```diff
@@ -82,6 +82,52 @@ if (this.something()) {
 
 For more details, check out [_Replacing the `switch` statement for object literals_](https://toddmotto.com/deprecating-the-switch-statement-for-object-literals/).
 
+### Single `return` statement
```

Comment:
this entire section sholud go under https://github.com/eventbrite/javascript/tree/master/general#multiple-if-else as case study

because the single return statement is a rule in `eslint` and sholuld already be covered but the tool